customer-onboarding-showcase/stack/c7/docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+# Use this only in dev environments. It's not intended for production usage.
+version: '3'
+services:
```

Comment:
Jup, they are usually the same but they can differ. (e.g. FormsFlow). 
To avoid that a user needs to find the right stack for the example they want to try out I would suggest having a stack folder in each example module.